xchange-stream-binance/src/main/java/info/bitrich/xchangestream/binance/futures/BinanceFuturesStreamingAccountService.java

```diff
@@ -0,0 +1,98 @@
+package info.bitrich.xchangestream.binance.futures;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import info.bitrich.xchangestream.binance.BinanceUserDataStreamingService;
+import info.bitrich.xchangestream.binance.dto.BaseBinanceWebSocketTransaction;
+import info.bitrich.xchangestream.binance.futures.dto.AccountUpdateBinanceWebsocketTransaction;
+import info.bitrich.xchangestream.core.StreamingAccountService;
+import info.bitrich.xchangestream.service.netty.StreamingObjectMapperHelper;
+import io.reactivex.rxjava3.core.Flowable;
+import io.reactivex.rxjava3.disposables.Disposable;
+import io.reactivex.rxjava3.processors.BehaviorProcessor;
+import io.reactivex.rxjava3.processors.FlowableProcessor;
+import org.knowm.xchange.currency.Currency;
+import org.knowm.xchange.dto.account.Balance;
+import org.knowm.xchange.exceptions.ExchangeException;
+import org.knowm.xchange.exceptions.ExchangeSecurityException;
+
+public class BinanceFuturesStreamingAccountService implements StreamingAccountService {
```

Comment:
See comment in BinanceFuturesStreamingTradeService regarding generic class.
I've noticed two changes from BinanceStreamingTradeService :
- Use of AccountUpdateBinanceWebsocketTransaction instead of OutboundAccountInfoBinanceWebsocketTransaction
- Use of ACCOUNT_UPDATE in line 69 instead of OUTBOUND_ACCOUNT_INFO

Can't we define a generic BaseBinanceStreamingTradeService\<T> and have the two services extend it with the right class? We can override the  openSubscriptions method to handle ACCOUNT_UPDATE  issue.